packages/manager/src/features/Volumes/VolumeCreate/CreateVolumeForm.tsx

```diff
@@ -273,9 +279,20 @@ const CreateVolumeForm: React.FC<CombinedProps> = (props) => {
                   data-qa-volume-size-help
                   variant="body1"
                 >
-                  A single Volume can range from 10 to {MAX_VOLUME_SIZE} GB in
-                  size and costs $0.10/GB per month. <br />
-                  Up to eight volumes can be attached to a single Linode.
+                  {dcSpecificPricing ? (
+                    <span>
+                      A single Volume can range from 10 to 10240 GB in size. Up
+                      to eight Volumes can be attached to a single Linode.
+                      Select a Region to see cost per GB
+                    </span>
+                  ) : (
+                    <span>
+                      A single Volume can range from 10 to {MAX_VOLUME_SIZE} GB
+                      in size and costs $0.10/GB per month. <br />
+                      Up to eight volumes can be attached to a single Linode.
```

Comment:
```suggestion
                      Up to eight volumes can be attached to a single Linode
```
![Screenshot 2023-08-21 at 2 05 05 PM](https://github.com/linode/manager/assets/114685994/6d8b2568-6f1d-4bef-8dd9-0e3cec63da93)

We've got a double period here: